R/metab_model_helpers.R

```diff
@@ -0,0 +1,90 @@
+#' Return the data types that may be used by metab_models using the 
+#' metab_model_interface.
+#' 
+#' Produces a unitted data.frame with the column names, units, and data format 
+#' to be used by metab_models that comply strictly with the 
+#' metab_model_interface.
+#' 
+#' Most models will require a subset of these data columns. Specialized models 
+#' may deviate from this format, but this is discouraged.
+#' 
+#' @return data data.frame with columns \itemize{
+#'   
+#'   \item{ \code{date.time} date-time values in solar time, in POSIXct format
+#'   with a nominal time zone of UTC.
+#'   
+#'   \item{ \code{DO.obs} dissolved oxygen concentration observations, \eqn{mg 
+#'   O[2] L^{-1}}{mg O2 / L}}
+#'   
+#'   \item{ \code{DO.sat} dissolved oxygen concentrations if the water were at 
+#'   equilibrium saturation \eqn{mg O[2] L^{-1}}{mg O2 / L}}. Calculate using 
+#'   \link{calc_DO_at_sat}}
+#'   
+#'   \item{ \code{depth} stream depth, \eqn{m}{m}}.
+#'   
+#'   \item{ \code{temp.water} water temperature, \eqn{degC}}.
+#'   
+#'   \item{ \code{light} photosynthetically active radiation, \eqn{\mu mol\ 
+#'   m^{-2} s^{-1}}{micro mols / m^2 / s}}
+#'   
+#'   }
+#'   
+#' @export
+#' @importFrom unitted u
+#' @examples
+#' mm_data()
+#' dplyr::select(mm_data(), depth, light)
+mm_data <- function() {
+  u(data.frame(
+    date.time= u(as.POSIXct("2050-03-14 15:9:27",tz="UTC"), NA), 
```

Comment:
i was thinking maybe we'd store units that way in the files, but then store them in their tz field within the POSIXct objects once they're in R. and then in that case, after read_ts then the data ought to have empty time units again. Thoughts?